### ISSUE TITLE
Powered Lamps are switchable

### ISSUE THREAD

**Issue by SmythRob** [iamceph/LampControl#6]
The plugin does not check if the lamp is powered and allows for it to be turned on / off even if powered.  Could you make it ignore powered lamps?

**Comment by PABessero** [comment]
I think it works by just changing the NBTs of the blocks, not sure if he could

**Comment by iamceph** [comment]
Yeah I can, but it was wroten this way by the original author. I'll make it configurable in next update that is coming :)
